src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -3815,5 +3815,40 @@ void verify(DefaultExpressionSyntax expr, PublicNullableAnnotation expectedAnnot
                 Assert.Equal(expectedState, info.FlowState);
             }
         }
+
+        [Fact]
+        public void GetTypeInfoOnNullableType()
+        {
+            var source = @"
+#nullable enable
```

Comment:
Is result going to be different in disable context?

Reply:
> Is result going to be different in disable context?

Please add test with disabled context

---
In reply to: [366369803](https://github.com/dotnet/roslyn/pull/40950#discussion_r366369803) [](ancestors = 366369803)